docs/usage.md

```diff
@@ -0,0 +1,62 @@
+# Usage Limits
+
+Each Altis environment is tailored to fit your usage, giving your customers a great experience while ensuring you only pay for what you need. Altis measures your traffic usage based on pages served to your users.
+
+Altis environments are designed to scale to meet demand dynamically within minutes, and usage is only monitored on a monthly basis.
+
+Your usage is available within the Altis Dashboard, and is updated throughout the day.
+## How traffic is measured
+
+Altis measures your usage based on requests made to the WordPress-based content management system measured as **Page Views**.
+## What is a page view?
+
+Page Views are any requests made to your WordPress-based content management system which invoke your application.
+
+Views are intended to count users who access your site, and exclude static content (such as CSS, JS, and images) and known bots.
+## How are views measured?
+
+Altis uses traffic statistics generated by the Amazon CloudFront CDN to measure views. HTTP requests which invoke the WordPress-based backend are counted as a view.
+
+For customers who are using their own CDN, similar statistics generated by the load balancer are used to measure views instead.
+## What traffic is excluded?
+
+Requests made by Altis automated systems and other known bots are excluded from views, even if they invoke the backend. Altis makes a best-effort attempt to exclude bot traffic, however cannot guarantee that all bots will be excluded. Altis also uses general firewall rules and best practices to block and mitigate malicious traffic.
+
+Enterprise environment tier customers may work with the Altis team to create custom firewall rules to block additional traffic.
+## Why don’t my page views match my web analytics (such as Altis Analytics or Google Analytics)?
+
+Analytics services including Altis Analytics and Google Analytics measure traffic using JavaScript in users’ browsers. This allows these tools to collect detailed visitor data including location, browser versions, and page view duration. These traffic counts can differ from Page Views for a number of reasons, including users who block JavaScript or run ad blockers, browser caches, and custom filtering within web analytics tools.
+
+Your billing usage is always based on page views measured by the Altis backend and not measured by other tools.
+## What data is collected for page view statistics?
+
+Page Views are only collected as an aggregate count and do not contain any personal information about your visitors.
```

Comment:
```suggestion
Page Views are collected as an aggregate count from the CDN access logs Altis stores. For further information, consult the Altis data processing addendum.
```

Reply:
Does there need to be a link to the "data processing adendum"? OR is it an adendum to their contract?

Reply:
It's part of the contract, yeah